registry/registry.go

```diff
@@ -99,7 +101,12 @@ func doRequest(req *http.Request, jar http.CookieJar, timeout TimeoutType, secur
 			return false
 		}
 
-		hostDir := path.Join("/etc/docker/certs.d", req.URL.Host)
+		if runtime.GOOS == "windows" {
+			hostDir = filepath.Join(os.TempDir(), "/docker/certs.d")
```

Comment:
We are still working out where exactly persistent storage should go. This is an interim change while all of that gets resolved and potentially daemon gets moved to a Windows service later - just has to be somewhere that will exist on Windows - /etc/ obviously doesn't